data/geolocation-cn

```diff
@@ -289,6 +290,7 @@ mmbang.info
 mmstat.com
 mogucdn.com
 mogujie.com
+moocollege.com
```

Comment:
In general, it is not necessary to submit domains that are not multi-homed. This domain only have one IP which is inside China. It doesn't matter which DNS server is used. Users will always get the "correct" IP.

Reply:
I see. 👌